eventhubs/client/azure-eventhubs/src/main/java/com/azure/eventhubs/EventReceiverOptions.java

```diff
@@ -78,8 +78,13 @@ public EventReceiverOptions beginReceivingAt(EventPosition position) {
      *
```

Comment:
nit:  Did you mean to close the `<p>` tag in the above comment?

Reply:
Good catch. I didn't want to leave that `<p>` hanging for too long. 🤣